bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/TestBufferedMutator.java

```diff
@@ -96,6 +101,36 @@ public void testBufferSizeFlush() throws Exception {
     }
   }
 
+  @Test
+  public void testBufferedMutatorWithNullAndEmptyValues() throws Exception {
+    BufferedMutatorParams params = new BufferedMutatorParams(sharedTestEnv.getDefaultTableName());
+    try (BufferedMutator bm = getConnection().getBufferedMutator(params)) {
+      Exception actualError = null;
+      try {
+        bm.mutate((List<? extends Mutation>) null);
+      } catch (Exception ex) {
+        actualError = ex;
+      }
+      assertNotNull(actualError);
+      actualError = null;
+
+      try {
+        bm.mutate(ImmutableList.<Mutation>of());
+      } catch (Exception ex) {
+        actualError = ex;
+      }
+      assertNull(actualError);
+      actualError = null;
+
+      try {
+        bm.mutate(new Put(new byte[0]));
+      } catch (Exception ex) {
+        actualError = ex;
+      }
+      assertNotNull(actualError);
+    }
+  }
```

Comment:
@igorbernstein2 Does this test case looks ok? OR should I include more scenarios around each operation?

Reply:
lgtm